lib/services/livesync/platform-livesync-service.ts

```diff
@@ -39,6 +43,10 @@ export abstract class PlatformLiveSyncServiceBase implements IPlatformLiveSyncSe
 			let canExecute = this.getCanExecuteAction(platform, appIdentifier);
 			let action = (device: Mobile.IDevice): IFuture<void> => {
 				return (() => {
+					const normalizePlatformName = this.$mobileHelper.normalizePlatformName(platform);
```

Comment:
@yyosifov I can introduce new method in platform-service and call it from here. 
I've tried to find the best place for tracking LiveSync operation and it seems this is the one - only the "full sync" will be tracked (the first operation that's executed when you call `tns run <platform>`. In case you break it and call it again, we'll track it again. In case you change any file during `tns run <platform>` is working, we'll not track it, as we've already sent data for it.